### PR TITLE
Block Fire Damage Variable

### PR DESCRIPTION
**Pull request by MEEPofFaith**
In case you want some blocks to take more or less damage from fire than usual.

### CODE REVIEW

**Comment by Quezler** [comment]
boop

File: core/src/mindustry/world/Block.java [hunk 1]
```diff
@@ -109,6 +109,8 @@
     public int health = -1;
     /** base block explosiveness */
     public float baseExplosiveness = 0f;
+    /** how much damage fire does per hit */
```
Comment (Quezler) [inline core/src/mindustry/world/Block.java:112]:
```suggestion
    /** how much damage fire does per hit. */
```

**Comment by Anuken** [comment]
This will cause conflicts with 7.0 fire code, so I can't merge this.

**Comment by Legomonster33** [comment]
stuff should take more fire damage if it has flammable in it, and less if it has coolants

**Comment by MEEPofFaith** [comment]
I can recreate a working non-conflicting version after 7.0 is merged.
Or you could do it yourself.

**Pull request closed without merging**